python_modules/libraries/dagster-dbt/dagster_dbt/utils.py

```diff
@@ -70,6 +70,29 @@ def __getattr__(self, item):
             else {}
         )
 
+        semantic_models = (
```

Comment:
Could you add a comment here with something similar to what you said in the PR description? Something like:

"Starting in dbt 1.8 semantic models must be defined using the SemanticModel class"